src/Fixer/PhpUnit/PhpUnitAttributesFixer.php

```diff
@@ -334,6 +337,7 @@ private static function fixDataProvider(Tokens $tokens, int $index, Annotation $
         }
 
         if (str_contains($matches[1], '::')) {
+            // @phpstan-ignore-next-line offsetAccess.notFound
```

Comment:
```suggestion
            // @phpstan-ignore offsetAccess.notFound
```

`@phpstan-ignore` automatically detects if the error is in the same or in the next line. I don't want to spam with similar comments, but it applies to multiple other places too.